common/app/templates/inlineJS/nonBlocking/interactiveFonts.scala.js

```diff
@@ -0,0 +1,7 @@
+var fonts = [].slice.apply(window.parent.document.styleSheets)
```

Comment:
Might wanna wrap this and resize in `iife` to avoid global scope? Maybe `try catch` to harden it.